client/src/components/Shop/Card/FilterButtonContainer/FilterButtonByOrder/FilterButtonByOrder.jsx

```diff
@@ -0,0 +1,32 @@
+import PropTypes from "prop-types";
+import { useNavigate, useLocation } from 'react-router-dom';
+
+function FilterButtonByOrder ({ dataCategory }) {
+
+    const location = useLocation();
+    const searchParams = new URLSearchParams(location.search);
+    const query = searchParams.get("query");
+
+    const queryContain = query && query.includes('all');
```

Comment:
ce fichier mériterait des commentaires aussi :)

Reply:
I have added a comment into this component :)